### PR TITLE
CDF-17596 Support timestamps with negative epoch values

### PR DESCRIPTION
**Pull request by TharinduDG**
- setting the start filter to the minimum possible epoch value to support negative timestamps

### CODE REVIEW

**Review by SunilVenekarKrishnamoorthy — APPROVED** [review]
👍

File: src/main/scala/cognite/spark/v1/PushdownUtilities.scala [hunk 1]
```diff
@@ -181,7 +181,7 @@ object PushdownUtilities {
       // Note that this way of aggregating filters will not work with "Or" predicates.
       Try(timestampLimits.filter(_.isInstanceOf[Min]).max).toOption // scalafix:ok
         .map(_.value)
-        .getOrElse(Instant.ofEpochMilli(0)),
+        .getOrElse(Instant.ofEpochMilli(-2208988800000L)),
```
Comment (vuhainguyen) [inline src/main/scala/cognite/spark/v1/PushdownUtilities.scala:184]: This constant can be imported from EpochMilli instead? 🤔

**Review by vuhainguyen — APPROVED** [review]
Nice 👍

File: src/main/scala/cognite/spark/v1/PushdownUtilities.scala (hunk 1 above)
Comment (vuhainguyen) [reply src/main/scala/cognite/spark/v1/PushdownUtilities.scala:184]: Alright maybe add a comment here to give a hint why/how do you find that number 😄

**Pull request merged**